pageserver/src/tenant/vectored_blob_io.rs

```diff
@@ -102,7 +102,22 @@ impl VectoredReadBuilder {
             start: start_offset,
             end: end_offset,
             blobs_at,
-            max_read_size,
+            max_read_size: Some(max_read_size),
+        }
+    }
+
+    #[cfg(test)]
```

Comment:
This is marked `cfg(test)`, but used in non test code.

Reply:
Ah, all of this is test code. Can still be removed though.

Reply:
yep, I'm waiting for the point that I integrate the whole k-merge thing into the compaction code, and at that time I can remove the test attributes.